projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -65,10 +68,41 @@ public static function get_instance() {
 	/**
 	 * Sets up class properties for REST API requests.
 	 */
-	public function rest_api_init() {
+	public function set_api_request() {
 		$this->is_api_request = true;
 	}
 
+	/**
+	 * Makes jetpack_admin_menu_link_destination available in users REST API endpoint.
+	 */
+	public function register_link_destination_meta() {
+		register_meta(
+			'user',
+			'jetpack_admin_menu_link_destination',
+			array(
+				'auth_callback' => array( $this, 'update_admin_color_permissions_check' ),
+				'description'   => __( 'Whether to use wp-admin links for unified admin menu.', 'jetpack' ),
+				'single'        => true,
+				'show_in_rest'  => array(
+					'schema' => array( 'default' => false ),
+				),
+				'type'          => 'boolean',
+			)
+		);
+	}
```

Comment:
@getdave Using the new approach to Atomic-syncing, we don't need to add this API data, is that correct?

Reply:
My current understand of this is no we do not need this any longer.

@mmtr For context, we are going to need to expose the link destination User setting from Dotcom to Atomic in order that we can address https://github.com/Automattic/wp-calypso/issues/47043. 

We were going to use the same approach as we did for syncing the admin color scheme.

What @obenland references above is him exposing the `link_destination` meta on the Users REST API endpoint. As we're no longer using REST to sync these user settings we should be ok to remove this now right?

Reply:
> As we're no longer using REST to sync these user settings we should be ok to remove this now right?
That's correct. We should now expose this data on the XMLRPC response of the `wpcom.getUser` endpoint and read it on wpcomsh.